azerothcore/azerothcore-wotlk
# fix(DB/Creature): Gyromasts Revenge

## Changes Proposed:
-  Move The Threshwackonator 4100 scripts from C++ to SAI
-  Correct the gossip menus that the creature uses
- Correct the script of the npc.


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4099



## Tests Performed:
- Ubuntu 20.04



## How to Test the Changes:

.go c id 6669
Right click the NPC and observe the menu
.q a 2078
Right click the NPC and observe the menu now
Click the gossip option and complete the quest to check the script.

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
